src/region/RegionCreator.tsx

```diff
@@ -77,6 +79,7 @@ export default function RegionCreator({ canDraw, className, onStart, onStop }: P
         const [x1, y1] = getPosition(x, y);
 
         setIsDrawing(true);
+        setIsReady(false);
```

Comment:
Which edge case/problem you're trying to solve by adding this new state?

Reply:
It avoids drawing the shape at 0,0 if there's only one position, such as with `isDrawing`.

Reply:
I'll add a check to avoid re-firing it each time the mouse moves.